runtime/doc/helphelp.txt

```diff
@@ -25,6 +25,9 @@ Help on help files					*helphelp*
 			The 'helplang' option is used to select a language, if
 			the main help file is available in several languages.
 
+			Help file sections are added to the |location-list|
+			and can be viewed with |:lwindow|.
```

Comment:
Are they added eagerly still? Or should this be changed to something like "To see the table of contents, type <M-]>."